### PR TITLE
proper position value in search injection

### PR DESCRIPTION
**Pull request by cfoellmann**
fix injection in search dropdown in core + allow DEFAULT_SEARCH_INTO_MODULE to work for the module

when dolibarr core fixes DEFAULT_SEARCH_INTO_MODULE this will allow the module to function as intended

### CODE REVIEW

**Comment by cfoellmann** [comment]
DEFAULT_SEARCH_INTO_MODULE  was fixed with https://github.com/Dolibarr/dolibarr/pull/14230

**Comment by cfoellmann** [comment]
PING 🏓

File: class/actions_searcheverywhere.class.php [hunk 1]
```diff
@@ -114,16 +114,16 @@ function addSearchEntry($parameters, &$object, &$action, $hookmanager) {
 			$modSearch = new modsearcheverywhere($db);
 
 			$this->results = array(
-				'searcheverywhere' => array(
-					'position' => $modSearch->numero,
-					'img'=>'object_searcheverywhere'
-					,'label'=>$langs->trans('Searcheverywhere')
-					,'text'=>img_picto('','object_searcheverywhere@searcheverywhere').' '.$langs->trans('Searcheverywhere')
-					,'url'=>dol_buildpath('/searcheverywhere/search.php',1).'?keyword='.urlencode($search_boxvalue)
+				'searchintosearcheverywhere' => array(
```
Comment (atm-john) [inline class/actions_searcheverywhere.class.php:117]: Is it compatible with Dolibarr 6 to 12 ?
Your fix in community is in develop (V13)

Comment (cfoellmann) [reply class/actions_searcheverywhere.class.php:117]: Did this ever work?
In the core code the `searchinto{$var}` seems to be old

**Pull request merged**